### PR TITLE
Add Filters, Location, Phone icons

### PR DESCRIPTION
**Pull request by Faradey27**

### CODE REVIEW

File: packages/wix-ui-icons-common/src/test/allicons.e2e.ts [hunk 1]
```diff
@@ -10,5 +10,5 @@ describe('All icons', () => {
 
   eyes.it('check icons with', async () => {
     await browser.wait(ExpectedConditions.presenceOf($(`[data-hook="icons-list"]`)));
-  });
+  }, {version: 'Add Filters, Location, Phone icons'});
```
Comment (shlomitc) [inline packages/wix-ui-icons-common/src/test/allicons.e2e.ts]: I say let's ditch this approach, let eyes fail and we will approve baseline before merge (until we will have applitools branch support)

Comment (Faradey27) [reply packages/wix-ui-icons-common/src/test/allicons.e2e.ts]: ok)

**Comment by lbelinsk** [comment]
@milkyfruit can we merge this?

**Comment by milkyfruit** [comment]
@lbelinsk yes.

**Pull request merged**